packages/polling-controller/src/PollingController.test.ts

```diff
@@ -28,14 +28,16 @@ describe('PollingController', () => {
         state: { foo: 'bar' },
       });
       controller.startPollingByNetworkClientId('mainnet');
+      jest.advanceTimersByTime(0);
+      await Promise.resolve();
```

Comment:
Would `await jest.advanceTimersByTimeAsync(0)` work without having to explicitly await for a promise?

``` suggestion
      jest.advanceTimersByTimeAsync(0);
```

Reply:
ooo I didn't know about this method! I've been doing what Jiexi did here for a while!

Reply:
we need to bump all of our jest usage

Reply:
indeed we should. But doesn't need to be in this PR

Reply:
this requires we update our version of jest. Which we should do, but perhaps in a subsequent PR?

Reply:
Ack, you're right, my bad. Okay, the old way is fine for now then.